packages/aws-cdk-lib/aws-ses/lib/dedicated-ip-pool.ts

```diff
@@ -2,6 +2,23 @@ import { Construct } from 'constructs';
 import { CfnDedicatedIpPool } from './ses.generated';
 import { IResource, Resource } from '../../core';
 
+/**
+ * What kind of scailing mode to use for this IP pool
+ *
+ * @see https://docs.aws.amazon.com/ja_jp/ses/latest/dg/dedicated-ip.html
```

Comment:
```suggestion
 * @see https://docs.aws.amazon.com/ses/latest/dg/dedicated-ip.html
```
Let's use the generic link (not everybody speaks Japanese 😄)